web/profiles/custom/os2loop/modules/os2loop_revision_date/config/install/system.action.extend_revision_date_one_month_action.yml

```diff
@@ -0,0 +1,11 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - os2loop_revision_date
+id: extend_revision_date_one_month_action
+label: 'Set next revision date to 1 month from now'
+type: node
+plugin: os2loop_revision_date_set_action
+configuration:
+  time_span: 2629743
```

Comment:
We usually don't put config into a modules `config/install` folder, but keep it all in `config/sync` (for now).

Reply:
Removed module config